source/includes/_08-additional-features.md.erb

```diff
@@ -8,7 +8,7 @@ In order to run a successful company data check a company must be uniquely
 identified. Some company data providers allow searching through the candidates 
```

Comment:
Add comma: "In order to run a successful company data check,"

Reply:
👍